# Номер камеры
uint8 camera_type

# Статус камеры
uint8 status